scripts/update/Updater.php

```diff
@@ -29,10 +30,18 @@ class Updater extends common_ext_ExtensionUpdater
      */
     public function update($initialVersion)
     {
-        $this->skip('1.0.0', '1.0.1');
+        $this->skip('1.0.0', '1.0.2');
 
-        if ($this->isVersion('1.0.1')) {
-            $this->setVersion('1.0.2');
+        if ($this->isVersion('1.0.2')) {
+            /** @var SecuritySettingsRepositoryInterface $settingsRepository */
+            $settingsRepository = $this->getServiceManager()->get(SecuritySettingsRepositoryInterface::SERVICE_ID);
+            $securitySettings = $settingsRepository->findAll();
+            $values = explode(PHP_EOL, $securitySettings->findContentSecurityPolicyWhitelist()->getValue());
+            $values = array_merge($values, ['https://getproctorio.com', ROOT_URL]);
+            $securitySettings->findContentSecurityPolicy()->setValue('list');
+            $securitySettings->findContentSecurityPolicyWhitelist()->setValue(implode(PHP_EOL, array_unique($values)));
+            $settingsRepository->persist($securitySettings);
+            $this->setVersion('1.0.3');
```

Comment:
```suggestion
            /** @var SecuritySettingsRepositoryInterface $settingsRepository */
            $settingsRepository = $this->getServiceManager()->get(SecuritySettingsRepositoryInterface::SERVICE_ID);
            
            $securitySettings = $settingsRepository->findAll();
            
            $values = explode(PHP_EOL, $securitySettings->findContentSecurityPolicyWhitelist()->getValue());
            $values = array_merge($values, ['https://getproctorio.com', ROOT_URL]);
            
            $securitySettings->findContentSecurityPolicy()->setValue('list');
            $securitySettings->findContentSecurityPolicyWhitelist()->setValue(implode(PHP_EOL, array_unique($values)));
            
            $settingsRepository->persist($securitySettings);
            
            $this->setVersion('1.0.3');
```

Do not you think is easier to read when we separate code by context? I would add this new lines for better readability